calico-vpp-agent/cni/network_vpp_routes.go

```diff
@@ -148,6 +148,20 @@ func (s *Server) UnroutePblPortsPodInterface(podSpec *storage.LocalPodSpec, swIf
 	}
 }
 
+func (s *Server) CreatePodRPFVRF(podSpec *storage.LocalPodSpec, stack *vpplink.CleanupStack) (err error) {
+	for _, ipFamily := range vpplink.IpFamilies {
+		vrfId, err := s.vpp.AllocateVRF(ipFamily.IsIp6, podSpec.GetVrfTag(ipFamily)+"-RPF")
```

Comment:
We should probably do a `podSpec.GetVrfTag(ipFamily, "-RPF")` as `podSpec.GetVrfTag()` has logic to limit the VRF name to the max VPP length.